src/ai/defaultai_warfare.cc

```diff
@@ -345,7 +345,7 @@ bool DefaultAI::check_enemy_sites(const Time& gametime) {
                            -3 :
                            0;
 				inputs[10] = (observer.second.mines_nearby == ExtendedBool::kTrue) ? 3 : -1;
-				inputs[11] = (observer.second.mines_nearby == ExtendedBool::kTrue) ? 0 : 0;
+				inputs[11] = (observer.second.mines_nearby == ExtendedBool::kTrue) ? 0 : 0;  // NOLINT
```

Comment:
this is definitly an error.
should read 
inputs[11] = (observer.second.mines_nearby == ExtendedBool::kTrue) ? 2 : 0;